src/main/java/com/autotune/analyzer/serviceObjects/BulkJobStatus.java

```diff
@@ -115,130 +108,310 @@ public void setExperiments(Map<String, Experiment> experiments) {
+            DateTimeFormatter formatter = DateTimeFormatter
+                    .ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
+                    .withZone(ZoneOffset.UTC);  // Ensure it's in UTC
+
+            return formatter.format(instant);
+        }
+
+        // Method to set a notification in the map
+        public void setNotification(String key, Notification notification) {
+            if (this.notifications == null) {
+                this.notifications = new HashMap<>(); // Initialize if null
+            }
+            this.notifications.put(key, notification);
+        }
+
+        public Map<String, Notification> getNotifications() {
+            return notifications;
+        }
+
+        public void setNotifications(Map<String, Notification> notifications) {
+            this.notifications = notifications;
+        }
 
-    public AtomicInteger getProcessed_experiments() {
-        return processed_experiments;
+        public BulkInput getInput() {
+            return input;
+        }
+
+        public void setInput(BulkInput input) {
+            this.input = input;
+        }
     }
 
-    public void setProcessed_experiments(int count) {
-        this.processed_experiments.set(count);
+    @JsonFilter("experimentFilter")
+    public static class API_Response {
+        private CreateAPIResponse create = new CreateAPIResponse();
+        private GenerateAPIResponse recommendations = new GenerateAPIResponse();
+
+        public CreateAPIResponse getCreate() {
+            return create;
+        }
+
+        public void setCreate(CreateAPIResponse create) {
+            this.create = create;
+        }
+
+        public GenerateAPIResponse getRecommendations() {
+            return recommendations;
+        }
+
+        public void setRecommendations(GenerateAPIResponse recommendations) {
+            this.recommendations = recommendations;
+        }
     }
 
-    // Utility function to format Instant into the required UTC format
-    private String formatInstantAsUTCString(Instant instant) {
-        DateTimeFormatter formatter = DateTimeFormatter
-                .ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
-                .withZone(ZoneOffset.UTC);  // Ensure it's in UTC
+    @JsonFilter("experimentFilter")
+    public static class CreateAPIResponse {
+        private KruizeResponse response;
+        private CreateExperimentAPIObject request;
 
-        return formatter.format(instant);
+        public KruizeResponse getResponse() {
+            return response;
+        }
+
+        public void setResponse(KruizeResponse response) {
+            this.response = response;
+        }
+
+        public CreateExperimentAPIObject getRequest() {
+            return request;
+        }
+
+        public void setRequest(CreateExperimentAPIObject request) {
+            this.request = request;
+        }
     }
 
+    @JsonFilter("experimentFilter")
+    public static class GenerateAPIResponse {
```

Comment:
Naming is confusing ... please name this as `GenerateRecommendationsAPIResponse` and rename `createAPIResponse` as `createExperimentAPIResponse`